### DISCUSSION TITLE
osu confuses the selected speedmod for a beatmap when starting a song directly upon start of the game

### DISCUSSION THREAD

**Discussion by XeZey** [ppy/osu#28889] [Q&A]
### Type

Game behaviour

### Bug description

This was my first session today. I started osu and immediately went to play this beatmap https://osu.ppy.sh/beatmapsets/1068145#taiko/2237541. Here osu was having low perfomance during the first seconds, as the game just being started up (Im not sure why or when this happens, as I tried reproducing this by restarting my pc and it did not seem to have this perfomance drop, so might be something else that I just dont know about) I remember having 0.5 (or 0.6? not entirely sure) with pitch selected at first but then I changed it to 0.75 default with shifted pitch before I started the map. I didnt notice anything immediately wrong, but at some random point within the beatmap (probably like 10 seconds in), the pitch, along with everything else, suddenly speeded up to what I would assume is 0.75. This means the beginning part was actually played on 0.5 the entire time, and this would also make sense to me as I have played this map many times before and it did seem unusually slow. (no perfomance issues during playing) I know that this was not part of the song, as I have played the map many times before. My internet was also not fully started yet and was still on the way to connecting, so that could also be a possible factor.

I have tried reproducing this but I cannot manage to do so as of now.
The logs that are attributed to this instance are most likely 1720948715, as this was the first time I have started the game today.

### Version

2024.625.2

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16216384/compressed-logs.zip)

**Comment by bdach** [comment]
Moving to discussions due to lack of reproduction scenario
